yt_dlp/extractor/nbc.py

```diff
@@ -208,16 +211,15 @@ class NBCSportsVPlayerIE(InfoExtractor):
 
     @staticmethod
     def _extract_url(webpage):
-        iframe_m = re.search(
-            r'<(?:iframe[^>]+|div[^>]+data-(?:mpx-)?)src="(?P<url>%s[^"]+)"' % NBCSportsVPlayerIE._VALID_URL_BASE, webpage)
-        if iframe_m:
-            return iframe_m.group('url')
+        video_urls = re.search(
+            r'(var videoSrc = |div[^>]+data-(?:mpx-)?src=)"(?P<url>%s[^\"]+)' % NBCSportsVPlayerIE._VALID_URL_BASE, webpage)
```

Comment:
match the old regex as well just in case

Reply:
done!